content/docs/faq-styling.md

```diff
@@ -8,6 +8,12 @@ category: FAQ
 
 ### How do I add CSS classes to components?
 
+Import the CSS file in which the styles are scoped:
+
+```jsx
+import from './path/to/Filename.css';
```

Comment:
This is doesn't work out of the box, it requires special loader for Webpack. Overall, it's probably best not to entangle Webpack in the FAQ unless absolutely necessary. I think mentioning importing CSS with `css-loader` as an alternative to CSS-in-JS below, however, would be appropriate, maybe mentioning CSS modules and their benefits (like being able to use SASS/LESS/PostCSS directly, interop with CSS tooling, etc.)

Reply:
Yes we can add Css modules instead.
There's a article on styling in React.May be we can add this link I found it useful:
https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822